.github/workflows/format-check.yml

```diff
@@ -0,0 +1,14 @@
+jobs:
+    dprint-check:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v3
+        - uses: actions/setup-node@v3
+        - run: $(npx dprint check | grep "Found * not formatted file*.") >> "$GITHUB_STEP_SUMMARY"
```

Comment:
Is this working around the large text output?

At the end of it, we'd probably want to not hide all of the output text.

Reply:
Yes, right now it swallows all output and puts only a _"found ..."_ line in the step summary. What would we want the rest of the output for, though?

Amusingly, that line might be coincidentally logged from a file inside the `dprint check` output. But the step summary is only checked for existence right now in https://github.com/DefinitelyTyped/dt-mergebot/pull/475. So it wouldn't matter if it's the "right" message.

Reply:
Removing this file per the _August 10th_ update in https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/65993.